rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "stable"
+components = []
+targets = []
+profile = "minimal"
```

Comment:
Why the `minimal` profile? It doesn't include `rustfmt` and `clippy`.

I'd remove the `components` and `profile` lines.